pkg/kdk/destroy.go

```diff
@@ -15,26 +15,25 @@
 package kdk
 
 import (
-	"context"
 	"fmt"
+
 	"github.com/Sirupsen/logrus"
-	"github.com/cisco-sso/kdk/internal/pkg/utils/simpleprompt"
+	"github.com/cisco-sso/kdk/pkg/prompt"
 	"github.com/docker/docker/api/types"
-	"github.com/docker/docker/client"
 )
 
-func Destroy(ctx context.Context, dockerClient *client.Client, logger logrus.Entry) error {
+func Destroy(cfg KdkEnvConfig, logger logrus.Entry) error {
 
 	var containerIds []string
 
-	containers, err := dockerClient.ContainerList(ctx, types.ContainerListOptions{})
+	containers, err := cfg.DockerClient.ContainerList(cfg.Ctx, types.ContainerListOptions{})
 
 	if err != nil {
 		logger.WithField("error", err).Fatal("Failed to list docker containers")
 	}
 	for _, container := range containers {
 		for _, name := range container.Names {
-			if name == "/"+KdkConfig.AppConfig.Name {
+			if name == "/"+cfg.ConfigFile.AppConfig.Name {
```

Comment:
any reason not to pad the `+` operator with spaces? It's rather hard to read this way.